docker/core/Dockerfile

```diff
@@ -1,29 +1,35 @@
+# FIXME use alpine
 FROM node:14
 
 RUN mkdir -p /usr/app/src \
 	&& mkdir -p /usr/app/media \
-	&& mkdir -p /usr/app/scripts
+	&& mkdir -p /usr/app/scripts \
+	&& apt-get update \
+  && apt-get install netcat -y
```

Comment:
I believe @serge1peshcoff mentioned that we do not need this anymore, and we could also remove the wait script